### PR TITLE
Adding RSS feed warning translation

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: src/locales_json/en/widget-rss.json [hunk 1]
```diff
@@ -38,5 +38,9 @@
   "title-format": {
     "heading": "Title Style",
     "text": "Title Style Sample Text"
+  },
+  "warning": {
+    "anchor": "here",
+    "message": "Please note that errors were detected in the feed URL provided that may effect how it is shown in your presentation. You can view a summary of the errors detected"
```
Comment (donnapep) [inline src/locales_json/en/widget-rss.json]: @stulees "effect" should be "affect".

**Comment by donnapep** [comment]
@stulees Small grammatical error. Otherwise :+1:

**Comment by stulees** [comment]
@donnapep Correct and rebased, thanks

**Pull request merged**